plugins/core/logreport.go

```diff
@@ -85,3 +100,88 @@ func (t *Tracer) ReportLog(ctx, timeObj interface{}, level, msg string, labels m
 
 	t.Reporter.SendLog(logData)
 }
+
+func (t *Tracer) GetLogContext(withEndpoint bool) interface{} {
+	var (
+		serviceName  string
+		instanceName string
+		endpoint     string
+
+		activeSpan TracingSpan = noopContext
+	)
+
+	if s, ok := t.ActiveSpan().(logTracingSpan); ok && s != nil {
```

Comment:
Please just cast to `TracingSpan`(using the original function code), The original logic has been verified in multiple milestones.